### ISSUE TITLE
Program log: Error: UninitializedAccount

### ISSUE THREAD

**Issue by cormacncheese** [metaplex-foundation/metaplex#1627]
When trying to buy an NFT from a public store I get the following transaction error in Solscan:

> Program log: Instruction: Revoke
> Program log: Error: UninitializedAccount
> Program Token Program  consumed 1313 of 200000 compute units
> Program returned error: instruction requires an initialized account

**Comment by neurofoo** [comment]
I'm getting what could be the same error. It happens with Instant Sales.

The devtools log console is

```
sendPlaceBid Error: Transaction failed: Error: UninitializedAccount
```

and the Solscan transaction log shows the same error as you. 

I think this is related to #1557 's fix for a bug.

**Comment by khoiuit99** [comment]
Any solution to fix this bug?

**Comment by neurofoo** [comment]
Answers on discord: https://discord.com/channels/848060988636921856/898624556460306493/934662988466651256


the critical piece seems to be 

```
overallInstructions.push([...instructions, ...cleanupInstructions.reverse()]);
```

in sendPlaceBid.ts

specifically, add the .reverse() call

**Comment by khoiuit99** [comment]
Oh yes, it works.
Thank you brother!

**Comment by neurofoo** [comment]
@cormacncheese looks like #1620 fixes this.

**Comment by samuelvanderwaal** [comment]
Should be fixed by #1620. If not, feel free to reopen.

**Comment by cormacncheese** [comment]
@neurofoo's solution fixed it for me!
